epsagon/events/urllib3.py

```diff
@@ -20,6 +20,9 @@
 )
 from ..utils import update_api_gateway_headers
 
+# Ignore AWS endpoints here since we capture botocore.vendored.requests
+IGNORE_PATTERNS = ['.amazonaws.com']
```

Comment:
So `urllib3` instrumentation is used only by `botocore`?
Because the purpose was not to filter requests to API-GW.
I'm confused.